base/src/org/compiere/process/BankStatementPayment.java

```diff
@@ -228,7 +228,7 @@ private MPayment createPayment (int invoiceId, int bPartnerId,
 		payment.setDescription(description);
 		//
 		if (invoiceId != 0) {
-			MInvoice invoice = new MInvoice (getCtx(), invoiceId, null);
+			MInvoice invoice = new MInvoice (getCtx(), invoiceId, get_TrxName());

```

Comment:
I just saw a change here but all class is changed, please review your head based on last commit of develop